modules/apps/account/account-service/src/main/java/com/liferay/account/model/impl/AccountEntryImpl.java

```diff
@@ -22,4 +34,45 @@
 	public AccountEntryImpl() {
 	}
 
+	public Group getAccountEntryGroup() {
+		DynamicQuery dynamicQuery = _getGroupDynamicQuery();
+
+		List<Group> groups = GroupLocalServiceUtil.dynamicQuery(dynamicQuery);
+
+		if (ListUtil.isEmpty(groups)) {
+			return null;
+		}
+
+		return groups.get(0);
```

Comment:
I think we can just call `GroupLocalServiceUtil.fetchGroup(long companyId, long classNameId, long classPK)` for this?

https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/service/GroupLocalServiceUtil.java#L488-L489